bot.py

```diff
@@ -321,6 +324,23 @@ async def listen_for_response(
                 return message
         return None
 
+    async def sync_commands(
+        self,
+    ) -> tuple[list[app_commands.AppCommand], dict[int, list[app_commands.AppCommand]]]:
+        logger.info("Beginning to sync commands ...")
+        global_cmds_synced = await self.tree.sync()
```

Comment:
You might want to add some sort of flag to only sync global commands when desired. Just FYI, this operation is kind of slow since it has to update the bot's profile across the entire Discord interface (and usually unnecessary, since most app commands are in guilds only).

Reply:
Do you want `!sync` to have a bool argument for syncing only guild or all commands? I am having a rather hard time getting a bool through to the command.

Reply:
> Do you want !sync to have a bool argument for syncing only guild or all commands?

Forgot to comment on an update, but this has been resolved.